src/Donations/Endpoints/ListDonations.php

```diff
@@ -330,4 +335,23 @@ protected function getWhereConditions(QueryBuilder $query): array
             $dependencies,
         ];
     }
+
+    /**
+     * @param string $date
+     *
+     * @return string
+     */
+    private function maybeExpandDate(?string $date, string $direction): ?string
+    {
+        if (empty($date) || strlen($date) > 3) {
```

Comment:
I'm not sure how I feel about this. Shouldn't we verify if the string is a valid date instead of checking the string length?

Reply:
@alaca Please, check 1302a3be40bb2de958d65d04de9b35cf6ab95b65. I think that's what you meant, right?